index.html

```diff
@@ -65,36 +66,36 @@ <h4 class="experience-title">Professional Experience</h4>
                     </section>
                 </main>
                 <aside class="contact-education-skills-box">
-                    <section class="contact-box">
-                        <h4 class="contact-title">Contact</h4>
-                        <section class="phone-box">
+                    <section class="contact-box center">
+                        <h4 class="contact-title center">Contact</h4>
+                        <section class="phone-box center">
                             <div class="phone-icon">
                                 <img src="phone-icon.svg" alt="phone">
                             </div>
                             <nav class="phone">
                                 <a href="(314) 174-7754" class="phone-details">(314) 174-7754</a>
                             </nav>
                         </section>
-                        <section class="mail-box">
+                        <section class="mail-box center">
                             <div class="mail-icon">
                                 <img src="mail-icon.svg" alt="mail">
                             </div>
                             <nav class="mail">
                                 <a href="john.maverick@gmail.com" class="mail-details">john.maverick@gmail.com</a>
                             </nav>
                         </section>
-                        <section class="linkedin-box">
+                        <section class="linkedin-box center">
                             <div class="linkedin-icon">
                                 <img src="linkedin-icon.svg" alt="linkedin">
                             </div>
                             <nav class="linkedin">
-                                <a href="linkedin.com/in/john-maverick/" class="linkedin-details">linkedin.com/in/john-maverick/</a>
+                                <a href="linkedin.com/in/john-maverick/" class="linkedin-details center">linkedin.com/in/john-maverick/</a>
```

Comment:
Let's say we are on a page `www.mikolaj.pl`.

When we add a link like that:
```
<a href="about-me">About me</a>
```
you are creating a **relative** link.

It means that clicking on it redirects to
```
www.mikolaj.pl/about-me
```


You created a **relative** link as well. Clicking on it would redirect you from `www.mikolaj.pl` to
```
www.mikolaj.pl/linkedin.com/in/john-maverick/
```

Instead, you should do
```
<a href="https://linkedin.com/in/john-maverick/">
```

with `https://` at the beggining. That is an **absolute** link. Clicking on it would redirect you from `www.mikolaj.pl` to https://linkedin.com/in/john-maverick/